articles/azure/azs-ref-troubleshooting.md

```diff
@@ -41,18 +41,6 @@ When first logging in to the Azure Stack portal, you may see the following error
 
```

Comment:
Make sure you remove the link at the top of the article that points to this deleted section.

In fact, the list of links at the top may not really be necessary as there is an internal TOC provided on the right side of the page when the topic is rendered in the Knowledge Centre, so maybe just remove them all...

Reply:
I've removed the TOC as its pretty much pointless.